test/parallel/test-process-clobber.js

```diff
@@ -0,0 +1,7 @@
+/* eslint-disable no-global-assign */
+'use strict';
+
+require('../common');
+const assert = require('assert');
+
+assert.throws(() => process = null, /Cannot assign to read only property/);
```

Comment:
The built-in libraries are, this checks that `process` is not assignable per the second commit.